src/lib/y2security/autoinst_profile/security_policy_section.rb

```diff
@@ -42,6 +46,21 @@ def self.attributes
       #   @return [String,nil] SCAP action to apply on first boot ("none", "scan" or "remediate")
       # @!attribute policy
       #   @return [String,nil] Enabled policy
+
+      # Clones the security policy settings from the underlying system
+      def self.new_from_system
+        file = CFA::SsgApply.load
+        section = new
+        return section if file.empty?
+
+        section.action = if Y2Security::SecurityPolicies::Manager.instance.service_enabled?
+          (file.remediate == "yes") ? "remediate" : "scan"
```

Comment:
I would prefer here if it can be value per condition as it is more readable so something like:
```
section.action = if !Y2Security::SecurityPolicies::Manager.instance.service_enabled?
  "none"
elsif file.remediate == "yes"
  "remediate"
else
  "scan"
end
```
this way I found it more readable.